code/game/objects/items/door_seal.dm

```diff
@@ -0,0 +1,24 @@
+/obj/item/door_seal
+	name = "pneumatic seal"
+	desc = "A brace used to seal and reinforce an airlock. Useful for making areas inaccessible to those without opposable thumbs."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "pneumatic_seal"
+	lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items_righthand.dmi'
+	flags_1 = CONDUCT_1
+	resistance_flags = FIRE_PROOF | ACID_PROOF
+	force = 5
+	throwforce = 5
+	throw_speed = 2
+	throw_range = 1
+	w_class = WEIGHT_CLASS_BULKY
+	custom_materials = list(/datum/material/iron=2000)
+	attack_verb = list("clonked", "whacked", "bashed", "thunked", "battered", "bludgeoned")
+	var/seal_time = 3 SECONDS
+	var/unseal_time = 2 SECONDS
```

Comment:
these require dm documentation

Reply:
the vars?

Reply:
I think I documented the vars if that's what this was referring to, if not please let me know because I'm dumb